CHANGES.md

```diff
@@ -1,118 +1,164 @@
-1 Changelog <a id="h.17phlwhk50et"></a>
-===============
+1 Changelog
 
 1.1.1
 
 -   Improved the MEG landmark coordinates description.
--   Replaced ManufacturersCapModelName in meg.json with CapManufacturer
-   and CapManufacturersModelName.
+
+-   Replaced ManufacturersCapModelName in meg.json with CapManufacturer and
+CapManufacturersModelName.
+
 -   Remove EEGSamplingFrequency and ManufacturersAmplifierModelName
-   from the meg.json.
+from the meg.json.
+
 -   Improved the behavioural data description.
 
 1.1.0
 
--   Added support for MEG data (merged BEP008)
--   Added SequenceName field.
--   Added support for describing events with Hierarchical Event Descriptors [[8.5 Task events](#heading=h.daip42kp5ndz)].
--   Added VolumeTiming and AcquisitionDuration fields [[8.4 Task (including resting state) imaging data](#heading=h.r8mrcau3kkcq)].
--   Added DwellTime field.
+- Added support for MEG data (merged BEP008)
+- Added SequenceName field.
+- Added support for describing events with Hierarchical Event Descriptors [[4.3 Task events](https://bids-specification.readthedocs.io/en/latest/04-modality-specific-files/03-task-events.html)].
```

Comment:
isn't there a way to provide internal references so they work correctly in both html and pdf renderings? (I am not yet familiar with this doc building setup so do not have an answer ready myself)

Reply:
I can provide a relative path linking than including the URL. Readthedocs can handle that. I confirmed this implementation can work with the HTML rendering, but pdf rendering appears to still be an [open issue](https://github.com/bids-standard/bids-specification/issues/18)?

Reply:
update - I went through and changed the linking to the actual markdown files on github. Hopefully that should stabilize the references

Reply:
cool!  I did the same in my PR: https://github.com/bids-standard/bids-specification/pull/78/files#diff-d668661006216379ab2a5c5ff7061b8aR221

my concern though is the "fragility" of `#id` anchors.  If misspecified/renamed file would be detected by mkdocs, those `#id` could be mistyped and "the system" seems to not detect it :-(  dunno yet what (if anything) to do about it

Reply:
Nice! In respect to the linking, I think most of the filenames appear to be solid. For the linking I think it will be prone to some fragility because we still need to link to something. When I am building locally, mkdocs does give a warning if it sees a link that is not present (or if I gave an incorrect path to the file), perhaps giving a test to make sure that no warning exists? Though, I can't either think of a more robust solution that would be linking onto something more concrete (than the filenames).

Reply:
just for clarity: yes, it issues the warning if the file is misspecified, but it would not warn if the portion after `#` is incorrect. E.g. if you link to `04-modality-specific-files/01-magnetic-resonance-imaging-data.md#sequence-specificsTYPO` it wouldn't warn that there is incorrect `sequence-specificsTYPO`.
Indeed I do not thin we could achieve a better solution (besides adding some post-build ad-hoc testing) with this doc system -- just for comparison in sphinx references are more explicit, so build system would warn/fail if `:ref:` (possibly into within a subsection in a document) is wrong

Reply:
I see what you are saying. Yes the best solution I can think of too is the ad-hoc testing. I suppose for now with this system is to be careful regarding referencing subsections within a file and checking locally that the links work.